shop/templates/shop/product_detail.html

```diff
@@ -98,12 +99,65 @@ <h4 class="product-detail-name m-text16 p-b-13">
                 </div>
             </form>
             <br>
-            <div class="wrap-dropdown-content bo7 p-t-15 p-b-14">
-                <h5 class="js-toggle-dropdown-content flex-sb-m cs-pointer m-text19 color0-hov trans-0-4">
-                    {% trans "Reviews" %} (0)
-                    <i class="down-mark fs-12 color1 fa fa-minus dis-none" aria-hidden="true"></i>
-                    <i class="up-mark fs-12 color1 fa fa-plus" aria-hidden="true"></i>
-                </h5>
+            <div class="wrap-dropdown-content bo7 p-t-15 p-b-14 ">
+                <div class="wrap-dropdown-content bo7 p-t-15 p-b-14">
+					<h5 class="m-text19 trans-0-4">
```

Comment:
indent kiểu gì kỳ cục vậy em

Reply:
![image](https://user-images.githubusercontent.com/39004634/130720372-706f8d9a-1b40-4886-9805-44d08b668edb.png)
Trên máy em vẫn bình thường mà đẩy lên git lại bị như vậy ạ

Reply:
em check lại trên IDE nhé
VScode: Ctrl + a file hiện tại -> Click [Spaces] -> Select [Convert Indentation to spaces]
![image](https://user-images.githubusercontent.com/49477980/130722133-5ec901fe-e833-4fd2-9f43-80dba554bf72.png)